### DISCUSSION TITLE
Conflict of interests

### DISCUSSION THREAD

**Discussion by onlymarcus** [bisq-network/support#1536] [General]
It seems there may be a conflict of interest in the current BISQ arbitration process. Specifically, when I engage in significant transactions, notably those exceeding 0.5 BTC, a pattern emerges: I complete my part of the trade by sending XMR, but then the other party ceases communication, allowing the trade to escalate to arbitration, thereby forfeiting their 15% security deposit.

This recurrent behavior raises the question: Why would someone willingly lose 15% of their BTC? It's conceivable that there's a strategic motive at play, potentially one aimed at stifling competition within the XMR/BTC market. Data analysis might reveal that one or two unique onion addresses are consistently involved in these incidents, suggesting a deliberate attempt to deter competitors. A competitor, tired of the prolonged arbitration process and the challenge of liquidating the received BSQ on the BSQ/BTC market, might eventually abandon selling XMR on BISQ.

I urge the BISQ developers to scrutinize this issue and expedite a resolution. One possible solution could involve awarding the non-defaulting trader the security deposit of the party that disappeared. While there is an existing proposal (https://github.com/bisq-network/proposals/issues/421) addressing this issue, it has yet to be implemented. I believe addressing this will significantly enhance the trading environment on BISQ.

**Comment by pazza83** [comment]
I agree that https://github.com/bisq-network/proposals/issues/421 will help mitigate this issue.

Feel free to DM me the onion addresses of the user/s you have concerns with on Matrix. I think it is more likely the issue was caused by https://github.com/bisq-network/bisq/issues/6948

I have 3 large XMR/BTC traders that have experienced this issue recently. I think the issue is more prevalent when miner fees ramp up and trades get caught out in an unconfirmed state for an extended period. These circumstances can cause Bisq to report the trade as failed when it has not done so. 

> Why would someone willingly lose 15% of their BTC?

The amount of trades ending up in arbitration (both in number and bitcoin volume) have been decreasing since https://github.com/bisq-network/projects/issues/67 was implemented  around March 2023.

I do think there is a possibility of someone trying to manipulate things but am yet to see anything that looks like this is the case currently. However, the more that can be done that removes this possibility the better.
